packages/cozy-procedures/src/components/overview/helpers.js

```diff
@@ -0,0 +1,86 @@
+import formatDate from 'date-fns/format'
+import subMonths from 'date-fns/sub_months'
+import startOfMonth from 'date-fns/start_of_month'
+import pick from 'lodash/pick'
+
+/**
+ * createJsonFile - upload a json file containing the given data
+ *
+ * @param {string} filename - The json file name
+ * @param {string} destinationId - The id of the destination directory
+ * @param {string} content - The file content
+ * @return {object} Information about the jsonFile (id and name)
+ */
+export const createJsonFile = async (
```

Comment:
`createFileWithContent` ? It does not look like this method is specific to JSON files. It will be easier to move to cozy-client then.